### PR TITLE
Tranche staking strategy

### PR DESCRIPTION
**Pull request by ayush-jibrel**
Fixes # .

Changes proposed in this pull request:
- 
- 
-

### CODE REVIEW

File: src/strategies/tranche-staking/index.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'bonustrack';
```
Comment (bonustrack) [inline src/strategies/tranche-staking/index.ts:4]: This should be you GitHub username
```suggestion
export const author = 'ayush-jibrel';
```

**Review by bonustrack — COMMENTED** [review]
Can you remove the change from package-lock.json and fix the conflicts?

File: src/strategies/tranche-staking/index.ts (hunk 1 above)
Comment (ayush-jibrel) [reply src/strategies/tranche-staking/index.ts]: Done

**Comment by bonustrack** [comment]
@ayush-jibrel Sorry there is again a new conflict, can you fix it or just authorize maintainers to edit your PR so we can do it for you?

image

**Review by bonustrack — APPROVED** [review]
Ok perfect now!

**Pull request merged**